static/sass/_pattern_careers-progression.scss

```diff
@@ -0,0 +1,378 @@
+@import "../../node_modules/vanilla-framework/scss/patterns_icons";
+
+@mixin canonical-p-careers-progression {
+  .p-strip--careers-progression {
+    @extend %vf-strip;
+
+    background-color: #261f24;
+    color: vf-determine-text-color(#261f24);
+
+    &.is-hero {
+      padding-top: 16.5rem;
+
+      @media screen and (min-width: $breakpoint-small) and (max-width: 1112px) {
+        padding-top: 2.5rem;
+      }
+
+      @media screen and (max-width: ($breakpoint-small - 1)) {
+        padding-top: $sp-large;
+      }
+    }
+
+    .p-divider__block {
+      color: $color-x-light;
+
+      &::before {
+        background-color: rgba(255, 255, 255, 0.25);
+      }
+    }
+
+    .p-roles {
+      border-right: 1px solid rgba(255, 255, 255, 0.25);
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      justify-content: space-between;
+      padding-right: 1.8rem;
+
+      @media screen and (min-width: $breakpoint-small) and (max-width: $breakpoint-large) {
+        margin-top: 1rem;
+      }
+
+      @media screen and (max-width: $breakpoint-small) {
+        border: 0;
+      }
+
+      img {
+        border-radius: 0;
+      }
+    }
+
+    .p-strip--hero-title {
+      font-size: 6.625rem;
+      line-height: 6rem;
+      margin-bottom: 6rem;
+
+      @media screen and (min-width: $breakpoint-small) and (max-width: $breakpoint-large) {
+        font-size: 3.625rem;
+        line-height: 3rem;
+        margin-bottom: 3rem;
+      }
+
+      @media screen and (max-width: ($breakpoint-small - 1)) {
+        font-size: 2rem;
+        line-height: $sp-large;
+        margin-bottom: $sp-large;
+      }
+    }
+
+    // Reset Vanilla styles
+    video {
+      margin: 0;
+      padding: 0;
+      width: 100%;
+    }
+  }
+
+  // Custom animations
+  // @keyframes slide-right {
+  // 	0% {
+  // 		opacity: 0;
+  // 		transform: translate3d(-50%, -20%, 0);
+  // 	}
+  // 	50% {
+  // 		opacity: 0;
+  // 		transform: translate3d(-25%, -10%, 0);
+  // 	}
+  // 	100% {
+  // 		opacity: 1;
+  // 		transform: translate3d(0, 0, 0);
+  // 	}
+  // }
+
+  // .u-animation--slide-right {
+  // white-space: nowrap;
+  // display: block;
+  // overflow: hidden;
+  // margin: 3rem 0;
+  // animation: type 4s 1 forwards linear;
+  // }
+
+  @keyframes slide-right {
+    from {
+      opacity: 0;
+      transform: translateY(20%);
+      width: 0;
+    }
+
+    to {
+      opacity: 1;
+      transform: translateY(0%);
+      overflow: visible;
+      width: var(--width);
+    }
+  }
+
+  .u-animation--slide-right {
+    --width: 100%;
+
+    &-separator {
+      --width: 110px;
+    }
+
+    animation: slide-right map-get($animation-duration, slow) 1 forwards linear;
```

Comment:
The linear easing here feels very robotic.

Reply:
Added more custom `cubic-benzier` but let me know if you want something more specific https://cubic-bezier.com/